nssp/delphi_nssp/pull.py

```diff
@@ -70,14 +71,14 @@ def pull_nssp_data(socrata_token: str):
         results.extend(page)
         offset += limit
     df_ervisits = pd.DataFrame.from_records(results)
-    df_ervisits = df_ervisits.rename(columns={"week_end": "timestamp", 
-                                              "percent_visits_smoothed":"percent_visits_smoothed_combined",
-                                              "percent_visits_smoothed_1":"percent_visits_smoothed_influenza",})
+    print(df_ervisits.columns)
```

Comment:
guessing this is a debugging print statement?

Reply:
yup. removed. https://github.com/cmu-delphi/covidcast-indicators/pull/1952/commits/583e24eef7a6d7e75448976b89ab3d58e3e296dc